app/models/account.rb

```diff
@@ -92,7 +92,7 @@ class Account < ApplicationRecord
   validates :username, format: { with: /\A[a-z0-9_]+\z/i }, length: { maximum: 30 }, if: -> { local? && will_save_change_to_username? && actor_type != 'Application' }
   validates_with UnreservedUsernameValidator, if: -> { local? && will_save_change_to_username? }
   validates :display_name, length: { maximum: 30 }, if: -> { local? && will_save_change_to_display_name? }
-  validates :note, note_length: { maximum: 500 }, if: -> { local? && will_save_change_to_note? }
+  validates :note, length: { minimum: StaticSetting.last&.min_profile_description_character || 0, maximum: 500 }, if: -> { local? && will_save_change_to_note? }
```

Comment:
https://github.com/magicstone-dev/ecko/blob/db8d22534fa2986b011e2be08547d978a957dc32/spec/models/account_spec.rb#L595

Might trigger this spec error for this. Could you please check or run this spec 
RAILS_ENV=test rspec spec/models/account_spec.rb and run the spec. Here you might know if your code has added new expectations in test or changed the expectation as well.